### PR TITLE
Issue #9450: update example of AST for TokenTypes.IDENT

### PR DESCRIPTION
**Pull request by Binita-tech**
Closes: #9450

### CODE REVIEW

**Comment by Binita-tech** [comment]
please review this PR @romani

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -1703,6 +1703,22 @@
         JavaLanguageLexer.LITERAL_DOUBLE;
 
     /**
+     * <p>For example:</p>
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1706]: Move example after explanation as it is done in other javadoc

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 1 above)
Comment (Binita-tech) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1709]: Done with changes @romani

**Review by romani — APPROVED** [review]

**Pull request merged**